ajax/libs/meyer-reset/package.json

```diff
@@ -7,5 +7,10 @@
   "keywords": [
     "css",
     "reset"
-  ]
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/adamstac/meyer-reset"
+  },
+  "license": "MIT"
```

Comment:
I think this should not be `MIT`
License field in Readme.md shows
> None (public domain)

Reply:
So what it should be here
In my opinion "No license"=MIT

Reply:
Let's use `Unlicense`, refer http://unlicense.org/ 
> The Unlicense is a template for disclaiming copyright monopoly interest in software you've written; in other words, it is a template for dedicating your software to the **public domain**.

and https://en.wikipedia.org/wiki/Unlicense
> The Unlicense is a **public domain equivalent license** with a focus on an anti-copyright message

Reply:
> "No license"=MIT

I don't agree.

Reply:
The "Unlicense" license was updated